### PR TITLE
SDKS-8664: Added large segments telemetry values

### PR DESCRIPTION
**Pull request by javrudsky**
# iOS SDK

## What did you accomplish?
- Added large segments values to telemetry structs

### CODE REVIEW

File: Split/Telemetry/Storage/TelemetryConfig.swift [hunk 1]
```diff
@@ -11,13 +11,15 @@ import Foundation
...
     var mySegments: Int
+    var myLargeSegments: Int
     var impressions: Int
     var events: Int
     var telemetry: Int
 
     enum CodingKeys: String, CodingKey {
         case splits = "sp"
         case mySegments = "se"
+        case myLargeSegments = "ls"
```
Comment (EmilianoSanchez) [inline Split/Telemetry/Storage/TelemetryConfig.swift:22]: it is `mls` actually. We made a typo in the spec.

File: Split/Telemetry/Storage/TelemetryConfig.swift [hunk 2]
```diff
@@ -64,6 +68,8 @@ struct TelemetryConfig: Codable {
 
     enum CodingKeys: String, CodingKey {
         case operationMode = "oM"
+        case largeSegmentsEnabled = "lsE"
+        case waitForLargeSegments = "wls"
```
Comment (EmilianoSanchez) [inline Split/Telemetry/Storage/TelemetryConfig.swift:72]: At the moment, we are not including this config in the telemetry

**Review by EmilianoSanchez — APPROVED** [review]

File: Split/Telemetry/Storage/TelemetryConfig.swift (hunk 1 above)
Comment (javrudsky) [reply Split/Telemetry/Storage/TelemetryConfig.swift:22]: 👍 Thank you doc!

File: Split/Telemetry/Storage/TelemetryConfig.swift (hunk 2 above)
Comment (javrudsky) [reply Split/Telemetry/Storage/TelemetryConfig.swift:72]: 👍

Comment (EmilianoSanchez) [reply Split/Telemetry/Storage/TelemetryConfig.swift]: Sorry @javrudsky , `wls` field is Okey

**Pull request merged**